CRUD.c

```diff
@@ -0,0 +1,186 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define FILENAME "users.txt"
+
+// User structure
+typedef struct {
+    int id;
+    char name[40];
+    int age;
+} User;
+
+// Let's check whether the file exists or not
+void checkFile() {

```

Comment:
Give variable and function names which provide more context on how they are being used or what they do

Reply:
👍